### PR TITLE
ansible 2.1.1.0-1

### PR DESCRIPTION
**Pull request by jaydrogers**
- [X] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [X] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

Updated Ansible to official release of Ansible 2.1.1.0-1 from their Github

### CODE REVIEW

**Comment by dunn** [comment]
The test is failing: http://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/5510/; did they change the syntax?

File: Formula/ansible.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Ansible < Formula
   desc "Automate deployment, configuration, and upgrading"
   homepage "https://www.ansible.com/"
-  url "https://releases.ansible.com/ansible/ansible-2.1.0.0.tar.gz"
-  sha256 "403443b55f6522cf86f9b41544a9b6b81f7d27f088446e85334d5d7a6317ccad"
+  url "https://github.com/ansible/ansible/archive/v2.1.1.0-1.tar.gz"
```
Comment (dunn) [inline Formula/ansible.rb]: Let's use https://releases.ansible.com/ansible/ansible-2.1.1.0.tar.gz instead.

**Comment by jaydrogers** [comment]
Whoops, good call on using the `releases.ansible.com` server. I am able to replicate the failing as well.

It's failing on the tests due to core ansible modules missing. When I do a `brew install ansible`, it installs Ansible 2.1.1.0 without a hitch, but then all of the core modules of Ansible are not loading correctly (`setup`, `name`,  etc.)

If I install 2.1.1.0 via `pip`, the modules seem to load no problem -- so this means that Ansible didn't make any syntax changes.

Not sure why this would be happening though. Anyone else have any thoughts? Is it some sort of environment variable or reference to the modules directory that is missing?

**Comment by dunn** [comment]
I suppose it's possible there's something different about the GitHub tarball. Try replacing it and testing locally, just in case.

**Comment by samdoran** [comment]
@dunn @jaydrogers The GitHub releases don't include the modules, which are Git submodules.

I just tested using the following URL and hash and it works fine:

```ruby
url "https://releases.ansible.com/ansible/ansible-2.1.1.0.tar.gz"
sha256 "61e739c123923ba90169f42c54e5f51df759ed40b4e332a7160d7db963d5678b"
```

**Comment by dunn** [comment]
> The GitHub releases don't include the modules, which are Git submodules.

Good to know; thanks!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/86d2cce908d1253b41a1473cfa2d46a5cdb2fa0b. Thank you for your contribution to Homebrew @jaydrogers; we appreciate it! :smiley_cat:

**Pull request closed without merging**